### ISSUE TITLE
Version 2.9.2 not working

### ISSUE THREAD

**Issue by Froelund** [bugsnag/bugsnag-react-native#228]
### Description
Trying to upgrade to `react-native-bugsnag@^2.9.2`.

### Issue
My packager is complaining that `./NativeSerializer` could not be found from `Bugsnag.js`.
```
Error: Unable to resolve module `./NativeSerializer` from `/<project-path>/node_modules/bugsnag-react-native/lib/Bugsnag.js`: The module `./NativeSerializer` could not be found from `/<project-path>/node_modules/bugsnag-react-native/lib/Bugsnag.js`. Indeed, none of these files exist:
```

### Environment

Library versions:

```shell
── bugsnag-react-native@2.9.2
```

**Comment by Froelund** [comment]
I just found the issue - I believe. 
In [bugsnag-react-native/package.json](https://github.com/bugsnag/bugsnag-react-native/blob/v2.9.2/package.json):
`"prepublish": "mkdir -p lib && babel src/Bugsnag.js -s -o lib/Bugsnag.js"`
Should be
`"prepublish": "mkdir -p lib && babel --source-maps --out-dir='lib' --ignore='__tests__' src",`

**Comment by Froelund** [comment]
The problem is already fixed on master: https://github.com/bugsnag/bugsnag-react-native/pull/227

**Comment by fractalwrench** [comment]
Addressed in v2.9.3
